scripts/genunicodetables.lua

```diff
@@ -17,18 +17,22 @@
 --    which don't have ambiguous or double width, and emoji_all has all Emojis.
 if arg[1] == '--help' then
   print('Usage:')
-  print('  genunicodetables.lua UnicodeData.txt CaseFolding.txt ' ..
-        'EastAsianWidth.txt emoji-data.txt')
-  print('                       unicode_tables.generated.h')
+  print('  genunicodetables.lua unicode/ unicode_tables.generated.h')
   os.exit(0)
 end
 
-local unicodedata_fname = arg[1]
-local casefolding_fname = arg[2]
-local eastasianwidth_fname = arg[3]
-local emoji_fname = arg[4]
+local basedir = arg[1]:gsub('[/\\]$', '')
```

Comment:
> On *nix it is valid to have directory like /foo/bar\, so this gsub is generally not correct.

You're right.  We already know what the path separator is.  It should only be matching/replacing that.

> why you think it may receive …/? CMakeLists.txt does not contain such path.

CMakeLists.txt isn't the only way to call this.  I'm just trying to make sure it handles possible input correctly.  I could use penlight's pl.path.join, but adding another dependency seems heavy handed for this one thing.